### PR TITLE
[Feature] Add truncated normal weight init

### PR DESCRIPTION
**Pull request by zhouzaida**
Add `trunc_normal_init` and `TruncNormalInit`

Reference:

- https://github.com/pytorch/pytorch/blob/master/torch/nn/init.py#L153

### CODE REVIEW

**Comment by hellock** [comment]
There are some conflicts to be fixed.

**Comment by zhouzaida** [comment]
> There are some conflicts to be fixed.

got it

**Review by MeowZheng — COMMENTED** [review]
should we consider using kstest to check truncated normal?

**Comment by zhouzaida** [comment]
> Is the file modelA.pth necessary in this PR? If not, we should clean them.

deleted

File: tests/test_cnn/test_weight_init.py [hunk 1]
```diff
@@ -47,6 +47,16 @@ def test_normal_init():
     # TODO: sanity check distribution, e.g. mean, std
 
 
+def test_truncated_normal_init():
+    conv_module = nn.Conv2d(3, 16, 3)
```
Comment (ZwwWayne) [inline tests/test_cnn/test_weight_init.py:51]: Suggest merge this function with `test_trunc_normal_init()`

Comment (zhouzaida) [reply tests/test_cnn/test_weight_init.py:51]: got it

**Comment by zhouzaida** [comment]
> should we consider using kstest to check truncated normal?

yet, I use kstest to check truncated normal.

**Review by MeowZheng — APPROVED** [review]
LGTM

**Pull request merged**